dockeradapter/filterdockerstream.go

```diff
@@ -24,62 +24,98 @@ const (
 	stdWriterSizeIndex = 4
 )
 
+// frameHeader holds the stream type and payload size for a Docker stream frame
+type frameHeader struct {
+	streamType  int
+	payloadSize int
+}
+
 // FilterDockerStream reads the requested stream from *bufio.Reader, strips the
-// prefix headers added by Dockers stdWriter.Write(p []byte) and returns a slice
+// frameHeader added by Dockers stdWriter.Write(p []byte) and returns a slice
 // of strings for each line received (splitting on '/n').
 //
 // If the stream processes without error but no lines are found then it will
 // return an empty slice of strings.
 //
 // The following streamTypes are supported (from the Docker stdcopy package):
 //  - Stdin    - 0
-//  - Stdout   - 1
+//  - Stderr   - 1
```

Comment:
Hm.  Looks like from https://github.com/moby/moby/blob/master/pkg/stdcopy/stdcopy.go#L16-L25 that this should still be Stdout, right?